examples/demo/Extras/LED_display_qt.py

```diff
@@ -0,0 +1,79 @@
+#  Copyright (c) 2019, Enthought, Inc.
+#  License: BSD Style.
+
+"""
+This demo illustrates use of the LEDEditor for displaying numeric values
+using a simulated LED display control.
+"""
+
+from threading import Thread
+from time import sleep
+
+from traits.api \
```

Comment:
This is an old style of import line continuation that we no-longer prefer.  Current Enthought style prefers one of:

- All on one line:
  ```
  from traits.api import HasTraits, Instance, Int, Bool, Float
  ```
  if less than 79 chars; or

- using parenthesis to wrap
  ```
  from traits.api import (
      HasTraits, Instance, Int, Bool, Float
  )
  ```
  if 80 chars or more.

Reply:
Updated the imports.